controller/destination/server.go

```diff
@@ -26,9 +26,8 @@ type (
 )
 
 var (
-	dnsCharactersRegexp             = regexp.MustCompile("^[a-zA-Z0-9-]{0,63}$")
-	startsAndEndsWithAlphanumRegexp = regexp.MustCompile("^(([a-zA-Z0-9].*[a-zA-Z0-9])|[a-zA-Z0-9])$")
-	containsAlphaRegexp             = regexp.MustCompile("[a-zA-Z]")
+	dnsCharactersRegexp = regexp.MustCompile("^[a-zA-Z0-9_-]{0,63}$")
```

Comment:
I believe that an underscore can _technically_ only be the first character in a name. not a blocker, this seems good enough.  DNS is nuts.

Reply:
I'm wrong, per https://stackoverflow.com/questions/2180465/can-domain-name-subdomains-have-an-underscore-in-it#2183140